### PR TITLE
Readme fixes based on review.

### PR DESCRIPTION
**Pull request by sorvell**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -84,6 +84,11 @@ and renders declaratively using `lit-html`.
 
         ```npm i @polymer/lit-element```
 
+    1. Install the webcomponents polyfill. If you're developing a reusable package, this should be a dev dependency.
```
Comment (kevinpschaaf) [inline README.md]: "...which you load in your tests, demos, etc."

File: README.md [hunk 2]
```diff
@@ -208,7 +219,10 @@ element is not pending another update, and any code awaiting the element's
 
 ## Bigger Example
 
-```JavaScript
+Note, this example uses decroators to create properties. Decorators are a proposed
```
Comment (TimvdLippe) [inline README.md]: s/decroators/decorators/

File: README.md [hunk 3]
```diff
@@ -92,7 +97,7 @@ and renders declaratively using `lit-html`.
 
     1. Run the development server and open a browser pointing to its URL:
 
-        ```polymer serve```
+        ```polymer serve --npm```
```
Comment (kevinpschaaf) [inline README.md]: Should not be required if using names for dependencies.

File: README.md [hunk 4]
```diff
@@ -84,6 +84,11 @@ and renders declaratively using `lit-html`.
...
         ```npm i @polymer/lit-element```
 
+    1. Install the webcomponents polyfill. If you're developing a reusable package, this should be a dev dependency.
+
+        ```npm i @webcomponentsjs/webcomponents```
+
+
     1. Create an element by extending LitElement and calling `customElements.define` with your class (see the examples below).
 
     1. Install the Polymer CLI:
```
Comment (kevinpschaaf) [inline README.md]: @next not required

File: README.md [hunk 5]
```diff
@@ -84,6 +84,11 @@ and renders declaratively using `lit-html`.
 
         ```npm i @polymer/lit-element```
 
+    1. Install the webcomponents polyfill. If you're developing a reusable package, this should be a dev dependency.
+
+        ```npm i @webcomponentsjs/webcomponents```
```
Comment (kevinpschaaf) [inline README.md]: `@webcomponentsjs/webcomponents` --> `@webcomponents/webcomponentsjs`

File: README.md [hunk 6]
```diff
@@ -112,14 +117,20 @@ current properties to return a `lit-html` template result to render
...
 
     class MyElement extends LitElement {
 
-      @property({type: String})
-      mood = 'happy';
+      static get properties() {
+        mood: {type: String}
+      }
+
+      constructors() {
```
Comment (kevinpschaaf) [inline README.md]: `constructors` --> `constructor`

File: README.md [hunk 7]
```diff
@@ -112,14 +117,20 @@ current properties to return a `lit-html` template result to render
...
+  <script src="node_modules/@webcomponents/webcomponentsjs/webcomponents-bundle.js"></script>
   <script type="module">
     import {LitElement, html, property} from '@polymer/lit-element';
 
     class MyElement extends LitElement {
 
-      @property({type: String})
-      mood = 'happy';
+      static get properties() {
+        mood: {type: String}
```
Comment (kevinpschaaf) [inline README.md]: missing return
```
      static get properties() {
        return {
          mood: {type: String}
        };
      }
```

**Pull request merged**